industry_restaurant/demo/pos_order_line.xml

```diff
@@ -0,0 +1,199 @@
+<?xml version='1.0' encoding='UTF-8'?>
+<odoo noupdate="1">
+    <record id="pos_order_line_18" model="pos.order.line">
+        <field name="order_id" ref="pos_order_6"/>
+        <field name="product_id" ref="product_product_47"/>
+        <field name="full_product_name">Combo(Chicken Burger + Coke)</field>
```

Comment:
Isn't it automatically retrieved from the product name, like in sale or purchase?

Reply:
no they are not retrieved

Reply:
OK :-/ just add a blank space before the parenthesis

Reply:
please add a blank space before the parenthesis